Added professors from the University of Basel

alucchi opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [ ] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [ ] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [ ] Combine multiple updates to a single institution into a **single PR.**

- [ ] Only submit one pull request per institution.

- [ ] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [ ] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [ ] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [ ] Check to make sure that you have no spaces after commas, or any missing fields.

- [ ] Check to make sure the home page is correct.

- [ ] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [ ] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [ ] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

alucchi committed:
Update csrankings-a.csv

Adding professors at the University of Basel

alucchi committed:
Update csrankings-i.csv

Adding prof. from University of Basel

emeryberger commented:
I've added the University of Basel; please read and check the instructions next time:

> NOTE: IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
> You must read and check all the boxes below by filling them in with an X or your PR will be rejected.
>

alucchi commented:
Thank you for doing so. I'm sorry, I did read the instructions but I
couldn't figure out how to add an institution. I added it
in country-info.csv (the instructions said we should edit this file, and
not other files) but my commit failed. Can I now resubmit my changes to the
files to the csrankings-[a-z].csv files?

Best regards,
Aurelien Lucchi

Processing Aurélien Lucchi,University of Basel,
https://omls.dmi.unibas.ch/en/persons/aurelien-lucchi/,V1ONSgIAAAAJ
78
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:79>
ERROR: This institution (University of Basel) was not found in
`institutions.csv`.
79
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:80>
Checking
https://dblp.org/search/author/api?q=author%3AAurelien%20Lucchi:$%3A&format=json&c=10
80
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:81>
Checking homepage URL (
https://omls.dmi.unibas.ch/en/persons/aurelien-lucchi/)
81
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:82>SUCCESS:
Page loaded successfully with requests.
82
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:83>***Test
failure for Aurelien Lucchi***.
83
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:84>Processing
Christian Tschudin,University of Basel,
https://dmi.unibas.ch/en/persons/christian-tschudin/,NOSCHOLARPAGE
84
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:85>
ERROR: This institution (University of Basel) was not found in
`institutions.csv`.
85
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:86>
Checking
https://dblp.org/search/author/api?q=author%3AChristian%20Tschudin:$%3A&format=json&c=10
86
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:87>
Checking homepage URL (https://dmi.unibas.ch/en/persons/christian-tschudin/)

87
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:88>SUCCESS:
Page loaded successfully with requests.
88
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:89>***Test
failure for Christian Tschudin***.
89
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:90>Processing
Florina M. Ciorba,University of Basel,
https://dmi.unibas.ch/en/persons/florina-m-ciorba/,NOSCHOLARPAGE
90
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:91>
ERROR: This institution (University of Basel) was not found in
`institutions.csv`.
91
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:92>
Checking
https://dblp.org/search/author/api?q=author%3AFlorina%20M%20Ciorba:$%3A&format=json&c=10
92
<https://github.com/emeryberger/CSrankings/actions/runs/11518225958/job/32064524107?pr=7863#step:6:93>
Checking homepage URL (https://dmi.unibas.ch/en/persons/florina-m-ciorba/)

On Fri, Oct 25, 2024 at 3:15 PM Emery Berger ***@***.***>
wrote:

> I've added the University of Basel; please read and check the instructions
> next time:
>
> NOTE: IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE
> SUMMARILY REJECTED.
> You must read and check all the boxes below by filling them in with an X
> or your PR will be rejected.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emeryberger/CSrankings/pull/7862#issuecomment-2437751259>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACL2IILRG62JYPTSEKDSJTZ5I765AVCNFSM6AAAAABQTDHQHKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZXG42TCMRVHE>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

emeryberger commented:
Just go ahead and open a new PR with those entries.

alucchi commented:
Done. I fixed the entry you suggested.

On Fri, Oct 25, 2024 at 4:14 PM Emery Berger ***@***.***>
wrote:

> Just go ahead and open a new PR with those entries.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emeryberger/CSrankings/pull/7862#issuecomment-2437924358>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACL2IKD6ICBUEOD7COE3QLZ5JG25AVCNFSM6AAAAABQTDHQHKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZXHEZDIMZVHA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>
